packages/core/src/table/table.scss

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+// TODO Fix styles so they work even if you don't explicitly use thead/tbody
+
+@import './../styles/tokens/generated/index';
+
+[cds-table] {
+  --background: #{$cds-alias-object-container-background};
+  --background-tint: #{$cds-alias-object-container-background-tint};
+  --background-shade: #{$cds-alias-object-container-background-shade};
+  --color: #{$cds-global-typography-color-400};
+  --border-radius: #{$cds-alias-object-border-radius-100};
+  --border-width: #{$cds-alias-object-border-width-100};
+  --header-separator-width: #{$cds-alias-object-border-width-100};
+  --border-color: #{$cds-alias-object-border-color};
+  --border-tint: #{$cds-alias-object-border-color-tint};
+  --cell-padding-vertical: #{$cds-global-space-5};
+  --cell-padding-horizontal: #{$cds-global-space-6};
+  --font-size: #{$cds-global-typography-body-font-size};
+}
+
+// Styles and default bordering
+[cds-table] {
+  background-color: var(--background);
+  border-collapse: separate;
+  border-spacing: 0;
+  color: var(--color);
+  margin: 0;
+  max-width: 100%;
+  width: 100%;
+
+  td,
+  th {
+    padding: var(--cell-padding-vertical) var(--cell-padding-horizontal);
+    text-align: center;
+    vertical-align: top;
+  }
+
+  th {
+    color: var(--color);
+    background-color: var(--background-shade);
```

Comment:
Looking at the demo it looks like maybe this should should be the background tint?